### DISCUSSION TITLE
Help getting rmView Running on PixelBook through Crostini Container

### DISCUSSION THREAD

**Discussion by tractatus-86** [bordaigorl/rmview#123] [Q&A]
So I've been trying to get the rmView to run through the Crostini Linux container using screenshare.

Versions
- remarkable 2 version 2.11.0.442
- latest VNC branch of the rmView

I have 5901 UDP port forwarded and opened through crostini and the network firewall.
I have 5900 TCP port forwarded and opened as well.

It appears that rmView recognizes the tablet and the remarkable 2.
I am able to SSH into the tablet through the crostini container as well so it appears that the devices can see each other.

However when the rmView starts all I see is 
![Screenshot 2021-12-21 10 27 21](https://user-images.githubusercontent.com/6187685/146955599-beeec052-3070-46a9-a552-5dcef50d10d8.png)

And here are the verbose logs

![Screenshot 2021-12-21 10 29 46](https://user-images.githubusercontent.com/6187685/146956002-71208919-79f8-420e-b8d8-069789d0444e.png)

Not sure what I maybe missing. 
I had a feeling that maybe the ports are not configured for the correct machines thats currently both UDP and TCP are simply configured for the Chromebook on the firewall.

**Comment by bordaigorl** [comment]
It does look like it's not communicating properly on UDP 5901.
If I had to guess there's something wrong in the port forwarding/firewall setup.
Also: make sure you start the screen share on the tablet or it would not start the challenge protocol.
